src/detectEyes.py

```diff
@@ -1,27 +1,141 @@
 import cv2
+from enum import Enum
 
-def detectEyes(image, detector):
-    '''Returns the coordinates of the eyes in the image.'''
+def centerCoordinates(f):
```

Comment:
```suggestion
def center_coordinates(f):
```
Based on the Google `.pylintrc`, we should be following a snake_case naming scheme across our code.

Reply:
I'm not a big fan of snake_case since typing an underscore is a lot more cumbersome than simple capitalization in camelCase. It also makes it harder to read in my opinion.

Reply:
Okay, that's fair enough - I'm down to use camelCase. If that's the case though, we should either adjust the pylintrc or remove it entirely so we don't get flooded with warnings.

Reply:
Isn't snake case the standard in python though? https://peps.python.org/pep-0008/